kernel/src/hil/aes.rs

```diff
@@ -0,0 +1,14 @@
+use returncode::ReturnCode;
+
+pub trait AESDriver {
```

Comment:
Crypto? I don't what your intentions are in future but you may want other crypto stuff as hash functions and so on? If not I suggest SymmetricEncryption